amazon-kinesis-client/src/test/java/software/amazon/kinesis/config/KCLAppConfig.java

```diff
@@ -236,6 +243,15 @@ private StsAsyncClient buildStsAsyncClient(AwsCredentialsProvider creds) {
                 .build();
     }
 
+    public final DynamoDbClient buildDynamoDbClient() throws IOException {
```

Comment:
Can't we use the async client defined in this class?

Reply:
Will address this with follow-up commits.